lib/cucumber-api/steps.rb

```diff
@@ -27,10 +27,39 @@
   end
 end
 
+Then (/^I grab "([^"]+)"$/) do |k|
```

Comment:
Precisely. It infers the JSON path from the name passed. I've changed these to "When" steps